src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -574,11 +574,23 @@ private protected unsafe TransformerEstimatorSafeHandle CreateTransformerFromEst
                 }
                 else
                 {
-                    fixed (byte* dataRootDir = Encoding.UTF8.GetBytes(AppDomain.CurrentDomain.BaseDirectory + char.MinValue))
+                    byte[] dataRoot;
+
+                    if (Directory.Exists(AppDomain.CurrentDomain.BaseDirectory + "/Data/DateTimeFeaturizer"))
+                    {
+                        dataRoot = Encoding.UTF8.GetBytes(AppDomain.CurrentDomain.BaseDirectory + char.MinValue);
+                    }
+                    else
+                    {
+                        dataRoot = Encoding.UTF8.GetBytes(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) + char.MinValue);
```

Comment:
Does this statement work all the time? would it make sense to get rid of the above?

Reply:
This is to be able to find the data files in NimbusML whether it was compiled with .Net core or the full .Net framework. both those cases are hit, just depends on the framework.

We have a lot of the featurziers work on a private branch because it wont be checked into master, and this fix was in that private branch. I thought it was also in master. If it was in master, then this is a regression. If it wasn't in master, then this is a needed fix for NimbusML.